app/constants.py

```diff
@@ -0,0 +1,5 @@
+global_admin = "Global Admin"
```

Comment:
Here are the defined names on the PermissionType - Data spreadsheet
https://docs.google.com/spreadsheets/d/1x_zZ8JLS2hO-zG0jUocOJmX16jh-DF5dccrd_OEGNZ0/edit?pli=1&gid=1448710921#gid=1448710921

adminGlobal
adminVrms
adminBrigade
adminProject
practiceLeadProject
memberProject
memberGeneral